bundler/src/main.rs

```diff
@@ -265,7 +288,6 @@ async fn bundle_endpoint(
     State(current_bundle): State<CurrentBundle>,
     if_none_match: Option<TypedHeader<IfNoneMatch>>,
 ) -> impl IntoResponse {
-    tracing::info!(counter.bundle_requests = 1);
     let etag = ETag::from_str(&format!(
```

Comment:
Since we saw the OPA instance bouncing between two ETAGs, can we log the requested/current etag here too?

Reply:
Good idea, adding